components/board/digital_interrupts.go

```diff
@@ -96,13 +85,6 @@ type BasicDigitalInterrupt struct {
 	pp  PostProcessor
 }
 
-// Config returns the config used to create this interrupt.
-func (i *BasicDigitalInterrupt) Config(ctx context.Context) (DigitalInterruptConfig, error) {
-	i.mu.RLock()
-	defer i.mu.RUnlock()
-	return i.cfg, nil
-}
-
```

Comment:
is this just one of those unused functions you mentioned and that's why it's removed?

Reply:
Yup, this is never used.